index.bs

```diff
@@ -704,35 +704,85 @@ During the above process, the user agent SHOULD show some UI to the user to guid
-    to locate it later for use, and also contains metadata that can be used by the [=[RP]=] to assess the strength of the
-    credential during registration.
-
-    The <dfn>clientDataJSON</dfn> attribute contains the [=JSON-serialized client data=] (see [[#cred-attestation]]) passed to
-    the authenticator by the client in order to generate this credential. The exact JSON serialization must be preserved as the
-    [=hash of the serialized client data=] has been computed over it.
-
-    The <dfn>attestationObject</dfn> attribute contains an [=attestation object=]. The contents of this object are
-    determined by the [=attestation statement format=] used by the authenticator. This object is opaque to, and
-    cryptographically protected against tampering by, the client. It contains the credential's unique identifier, [=credential
-    public key=], and attestation statement. It also contains any additional information that the [RP]'s server requires to
-    validate the attestation statement, as well as to decode and validate the bindings of both the client and authenticator
-    data. For more details, see [[#cred-attestation]].
+The {{AuthenticatorAttestationResponse}} interface represents the [=authenticator=]'s response to a client's request
+to create a new [=scoped credential=]. It contains information about the credential that can be used to identify it
+for later use, and metadata that can be used by the [=[RP]=] to assess the characteristics of the credential during
+registration.
+
+<pre class="idl">
+    [SecureContext]
+    interface AuthenticatorAttestationResponse : AuthenticatorResponse {
+        readonly attribute ArrayBuffer attestationObject;
+    };
+</pre>
+<div dfn-type="attribute" dfn-for="AuthenticatorAttestationResponse">
+    :   {{AuthenticatorResponse/clientDataJSON}}
+    ::  This attribute, inherited from {{AuthenticatorResponse}}, contains the [=JSON-serialized client data=] (see
+        [[#cred-attestation]]) passed to the authenticator by the client in order to generate this credential. The
+        exact JSON serialization must be preserved, as the [=hash of the serialized client data=] has been computed
+        over it.
+
+    :   <dfn>attestationObject</dfn>
+    ::  This attribute contains an [=attestation object=]. The contents of the object are determined by the
+        [=attestation statement format=] used by the authenticator. The object is opaque to, and cryptographically
+        protected against tampering by, the client. It contains the credential's unique identifier, the [=credential
+        public key=], and the attestation statement. It also contains any additional information that the [RP]'s
+        server requires to validate the attestation statement, as well as to decode and validate the bindings of both
+        the client and authenticator data. For more details, see [[#cred-attestation]] as well as
+        [Figure 3](#fig-attStructs).
```

Comment:
might as well fix this entire parag now -- there's various issues with it. Here's a suggested rewrite:

This attribute contains an [=attestation object=], which is opaque to, and cryptographically protected against tampering by, the client. The [=attestation object=] contains both [=authenticator data=] and an attestation statement. The former contains the AAGUID, a unique credential ID, and the [=credential public key=]. The contents of the attestation statement are determined by the [=attestation statement format=] used by the [=authenticator=]. It also contains any additional information that the [RP]'s server requires to validate the attestation statement, as well as to decode and validate the [=authenticator data=] along with the [=JSON-serialized client data=]. For more details, see [[#cred-attestation]] as well as \[Figure 3]\(#fig-attStructs).